docs/swupd.1.rst

```diff
@@ -566,6 +627,43 @@ SUBCOMMANDS
 
             Only runs the verify operation on the os-core and vi bundles.
 
+    - `--extra-files-only`
+
+        Like --picky, but it only removes extra files. It omits checking
+        hash values, and for missing files, directories and/or symlinks.
+
+
+FILES
+=====
+
+/usr/share/defaults/swupd
+
+    Sometimes a set of flags is always used for one, or many swupd commands. The
+    ``swupd configuration file`` provides a convenient way of persistently define
+    these flags so they don't need to be specified every time a command is run.
+
+    The configuration file is an INI type of file that consists of sections, each led
+    by a [section] header, followed by key/value entries separated by a '=' character.
+    Note that there should be no whitespace between key=value. The configuration
+    file may include comments, prefixed by either the '#' or the ';' characters.
+
+    There can be one section for each swupd command (e.g. [bundle-add], [update], etc.)
+    and one for global options (e.g. [GLOBAL]). Global options can be specified in the
+    either in the GLOBAL section, in a command section, or in both. Global options
+    specified in the command section have higher precedence than those specified in the
+    GLOBAL section, so it is possible to define a GLOBAL option that will apply to all
+    swupd command except for that one overwritten in the command section.
+
+    A sample swupd configuration file can be found at this location (this file should not
+    be modified):
+    /usr/share/defaults/swupd
+
+    To use it, it should be copied to one of these locations where swupd reads the
+    configuration from (in that order):
+    - /etc/swupd
+    - /run/swupd
```

Comment:
/run is a tmpfs mounted fs on Clear so it's not a good place for a user to copy the config file. We read the config from /run, but shouldn't be used unless it's something temporary. we added that more for compatibility reasons.

Reply:
I got those files from this comment -> https://github.com/clearlinux/swupd-client/pull/958#issuecomment-504493148, so do you think we should only use /etc/swupd there?

Reply:
I think so. I wouldn't recommend users to copy config files to /run